src/languages/php.js

```diff
@@ -394,6 +393,42 @@ export default function(hljs) {
     ]
   };
 
+  const FUNCTION_INVOKE = {
+    relevance: 0,
+    match: [
+      /\b/,
+      // to prevent keywords from being confused as the function title
+      regex.concat("(?!fn\\b|function\\b|", normalizeKeywords(KWS).join("\\b|"), "|", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
+      IDENT_RE,
+      regex.concat(WHITESPACE, "*"),
+      regex.lookahead(/(?=\()/)
+    ],
+    scope: {
+      3: "title.function.invoke",
+    }
+  };
+  FUNCTION_INVOKE.contains = [
+    {
```

Comment:
Please extract into `PARAMS_MODE` as we do in so many other grammars.

Reply:
Something like this with the common part?
```
const PARAMS_MODE = [
              VARIABLE,
              LEFT_AND_RIGHT_SIDE_OF_DOUBLE_COLON,
              hljs.C_BLOCK_COMMENT_MODE,
              STRING,
              NUMBER
            ]

Reply:
No, move the whole begin: `(` end: `)` rule into named constant `PARAMS_MODE`...

Reply:
@joshgoebel Done